makefiles/info.inc.mk

```diff
@@ -13,7 +13,7 @@ info-objsize:
 	  "") SORTROW=4 ;; \
 	  *) echo "Usage: $(MAKE) info-objsize SORTROW=[text|data|bss|dec]" ; return ;; \
 	esac; \
-	echo -e '   text\t   data\t    bss\t    dec\t    hex\tfilename'; \
+	echo '   text\t   data\t    bss\t    dec\t    hex\tfilename'; \
```

Comment:
This needs to be a double quote to expand the backslash escaped chars.

Reply:
double-quotes didn't make a difference on macOS, I needed to pass `-e` otherwise `\t` are not interpreted but simply printed.

Reply:
`printf` also seems to be working. @smlng does this work better on mac?